components/compositing/windowing.rs

```diff
@@ -211,15 +228,21 @@ pub struct EmbedderCoordinates {
 impl EmbedderCoordinates {
     /// Get the unflipped viewport rectangle for use with the WebRender API.
     pub fn get_viewport(&self) -> DeviceIntRect {
-        DeviceIntRect::from_untyped(&self.viewport.to_untyped())
+        self.viewport.clone()
     }
 
-    /// Get the flipped viewport rectangle. This should be used when drawing directly
-    /// to the framebuffer with OpenGL commands.
-    pub fn get_flipped_viewport(&self) -> DeviceIntRect {
+    /// Flip the given rect.
+    /// This should be used when drawing directly to the framebuffer with OpenGL commands.
+    pub fn flipped_rect(&self, rect: &DeviceIntRect) -> DeviceIntRect {
```

Comment:
```suggestion
    pub fn flip_rect(&self, rect: &DeviceIntRect) -> DeviceIntRect {
```

In order to make the first word of the function a verb.

Reply:
Renamed to `flip_rect` in [8c01639](https://github.com/servo/servo/pull/31417/commits/8c01639b1993c3b6c5fa57c1fcfc412716cf6f05)